Example/Podfile.lock

```diff
@@ -1,5 +1,5 @@
 PODS:
-  - Colander (0.1.1):
+  - Colander (0.2):
```

Comment:
I would change this to 0.1.2

Reply:
I didn't change this myself, I just ran a `pod install` to get the example project building.  I think this means that colander already is at `0.2`?